IsoLib/libisomediafile/src/MovieFragmentAtom.c

```diff
@@ -34,7 +34,8 @@ static void destroy(MP4AtomPtr s)
 
   self = (MP4MovieFragmentAtomPtr)s;
   if(self == NULL) BAILWITHERROR(MP4BadParamErr)
-  DESTROY_ATOM_LIST_F(atomList);
```

Comment:
In this part child boxes are destroyed which should also include the traf box that you destroy manually in [MP4MovieFile.c](https://github.com/MPEGGroup/isobmff/pull/52/files#diff-8bf77f7b60fe611682baccdbff10ef42f3d1d653bd9fe5aeeac9434405fd6e7e)